truffle/contracts/Energy.sol

```diff
@@ -6,32 +6,39 @@ contract Energy {
     uint public unitPrice;
     uint public offeredAmount;
     mapping (address => uint) public remainingEnergy;
+    uint public boughtAmount;  // sum of remainingEnergy;
     address public seller;
-    address public master_addr;
+    bool public noUpdate;
```

Comment:
where is `noUpdate` used?

Reply:
during blockchain sync, a client would like to know whether a contract needs to be subscribed for event updates